_posts/2016/2016-04-20-jenadevs-docker-party.md

```diff
@@ -0,0 +1,72 @@
+
+Upfront, all necessary material and code for the workshops was hosted on the jenadevs [GitHub](https://github.com/jenadevs) account, which even offers the [complete presentation](https://github.com/jenadevs/jenadevs-meetup-001-docker-party).
+Great news: _All material of **[jenadevs](https://github.com/jenadevs)** is free and will always be open-source!_
+
+### Beginners Workshop
+
+The goal of the beginner workshop was to setup the Docker environment, learn the basic commands, run the workflow and become familiar with Docker Compose.
+The participants followed the [official Docker birthday tutorial](https://github.com/docker/docker-birthday-3/blob/master/tutorial.md), while the more experienced Docker users, Benjamin Nothdurft and Kay Abendroth, offered assistance.
+As kittens bustled around the beamer and laptop screens, the Docker newbies built simple web applications using the Python microframework [Flask](http://flask.pocoo.org/) – all running in a container, of course.
+
+### Advanced Workshop
+
+This workshop focused on writing your own Dockerfiles, run the containers and integration testing with [CircleCI](https://circleci.com). Bastian Klein prepared an empty custom [Magento](https://magento.com) `Dockerfile` and `Circle.yml` with comments left only, where the implementation code was completely removed.
+The attendes then had to figure out how the Dockerfile commands needed to be configured under the guidance of the workshop maintainer.
+
+### Special Workshop
+
+The most experienced Dockers gathered together in the special workshop, where the main focus was a bit shifted. We had prepared three worstation laptops. Andreas Grohman introduced CoreOS to the each participant. The goal was to setup a cluster environment with three nodes and run Docker on it.
+Everyone succeeded.
+Afterwards Christian Köhler gave an insight on how to configure Kubernetes with a short demonstration. He also distinctly outlined the advantages to Docker Swarm.
+
+## Summary 
+
+In retrospect the first event of [jenadevs](https://twitter.com/jenadevs) was a total success. Every participant had the possibilty to have fun, meet other devs and learn something new.
+Besides the beginners, which had the possibility to finish their well-documented tutoial at home, all participants could under the cautious eyes of our tutors finish all workshop tasks with happy results. So it was obvious that a lot of guests joined parts of the orga team in the local bar and just talked about the expierience in their developer life, their favorite editors and some nerdier stuff.
+
+Thanks to all attendees and ePages coworkers for making this happen!
+
+## Next Meetup
+
+The goal of the Softwerkskammer Jena is to establish a platform for regular meetings. So we already arranged the next Meetup! This time it is going to be a classical [Coderetreat](http://coderetreat.org) with a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) basics talk and demo upfront.
```

Comment:
And we already arranged the next meetup!

It will be a classical [Coderetreat](http://coderetreat.org) with a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) basics talk and demo upfront.